### PR TITLE
Add SD of users

### PR DESCRIPTION
**Pull request by KsenBantsevich**

### CODE REVIEW

File: ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/concepts/concept_user.scs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+concept_user
+<- sc_node_class;
+<= nrel_inclusion: 
+	subject;
+=> nrel_main_idtf:
+    [пользователь]
```
Comment (KovalM) [inline ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/concepts/concept_user.scs]: Fix formatting

File: ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/concepts/concept_user.scs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	...
+	(*
+	=> nrel_main_idtf:
+		[Опр. (пользователь)]
+		(* <- lang_ru;; *);
+		[Def. (user)]
+		(* <- lang_en;; *);;
+	<- definition;;
+	<= nrel_sc_text_translation:
+		...
```
Comment (KovalM) [inline ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/concepts/concept_user.scs]: What about constants?

File: ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/relations/nrel_login.scs [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+<- antireflexive_relation;
+<- asymmetric_relation;
+<- antitransitive_relation;
+=>nrel_main_idtf:
+	[логин*]
+	(* <- lang_ru;; *);
+	[login*]
+	(* <- lang_en;; *);
+=> nrel_first_domain: concept_user;
+=> nrel_second_domain: concept_token;
```
Comment (KovalM) [inline ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/relations/nrel_login.scs]: What is concept_token?

File: ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/relations/nrel_login.scs [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+		...
+		(*
+		-> rrel_example: 
+			[Логин* - бинарное отношение, связывающее пользователя и уникальную строку, предназначенную для авторизации.]
+			(* <- lang_ru;; *);;
+		*);
+		...
+		(*
+		-> rrel_example: 
+			[]
```
Comment (KovalM) [inline ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/relations/nrel_login.scs]: Add English

File: ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/relations/nrel_user_used_language.scs [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+		...
+		(*
+		-> rrel_example: 
+			[]
+			(* <- lang_en;; *);;
+		*);;
+	<= nrel_using_constants:
+		{
+		concept_user;
+        languages
```
Comment (KovalM) [inline ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/relations/nrel_user_used_language.scs]: fix formatting, add binary_relation constant

File: ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/relations/nrel_user_used_language.scs [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+		...
+		(*
+		-> rrel_example: 
+			[Используемый пользователем язык* - бинарное отношение, связывающее пользователя и язык, который он использует для общения.]
+			(* <- lang_ru;; *);;
+		*);
+		...
+		(*
+		-> rrel_example: 
+			[]
```
Comment (KovalM) [inline ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/relations/nrel_user_used_language.scs]: Add English translate

File: ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/section_subject_domain_of_users .scs [hunk 7]
```diff
@@ -0,0 +1,35 @@
+section_subject_domain_of_users
```
Comment (KovalM) [inline ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/section_subject_domain_of_users .scs]: I think we can use already exist SD subject_domain_developers_actions_knowledge_base_cooperation_hierarchy. We will discuss about using current state of this SD for our propose. And let's discuss @ShunkevichDV why know we use users roles as norole relation?

**Review by KovalM — CHANGES_REQUESTED** [review]

File: ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/relations/nrel_language_used_by_the_user.scs [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+		[Опр. (используемый пользователем язык*)] 
+		(* <- lang_ru;; *);
+		[Def. (language used by the user*)]
+		(* <- lang_en;; *);;
+	<-  definition;;
+	<= nrel_sc_text_translation: 
+		...
+		(*
+		-> rrel_example: 
+			[Используемый пользователем язык* - бинарное отношение, связывающее пользователя и язык, который он использует для общения.]
```
Comment (ShunkevichDV) [inline ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/relations/nrel_language_used_by_the_user.scs]: For communication with who?
In different systems user can use different languages, we should say about it, this language is only for this system

File: ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/relations/nrel_language_used_by_the_user.scs [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+			(* <- lang_ru;; *);;
+		*);
+		...
+		(*
+		-> rrel_example: 
+			[Language used by the user* is a binary relation that connects the user and the language that they use to communicate.]
```
Comment (ShunkevichDV) [inline ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/relations/nrel_language_used_by_the_user.scs]: Why *they*?

File: ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/relations/nrel_login.scs [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+		[Опр. (логин*)] 
+		(* <- lang_ru;; *);
+		[Def. (login*)]
+		(* <- lang_en;; *);;
+	<-  definition;;
+	<= nrel_sc_text_translation: 
+		...
+		(*
+		-> rrel_example: 
+			[Логин* - бинарное отношение, связывающее пользователя и уникальную строку, предназначенную для авторизации в системе.]
```
Comment (ShunkevichDV) [inline ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/relations/nrel_login.scs]: unique within what scope?

**Review by ShunkevichDV — CHANGES_REQUESTED** [review]

**Review by ShunkevichDV — APPROVED** [review]

**Review by KovalM — APPROVED** [review]

**Pull request merged**